enterprise/api/v1/views/enterprise_customer_support.py

```diff
@@ -52,47 +53,116 @@ def paginate_queryset(self, queryset, request, view=None):
 
 class EnterpriseCustomerSupportViewSet(EnterpriseReadOnlyModelViewSet):
     """
-    API views for the ``enterprise-user`` API endpoint.
+    API views for the ``enterprise-customer-support`` API endpoint.
     """
-    queryset = models.EnterpriseCustomerUser.objects.all()
-    filter_backends = (DjangoFilterBackend,)
+    queryset = models.PendingEnterpriseCustomerUser.objects.all()
```

Comment:
[non-blocking][question]: curious why we updated the model here to pending enterprise customer user?

Reply:
I needed to expose the 'user_email' as a searchable attribute, but EnterpriseCustomerUser model does not a  direct model field for `user_email`, but has it as a property. Django required the search field to be a model field, so I had to switch to PendingEnterpriseCustomerUser which does have `user_email` as a model field. 

It does not impact the query results since the retrieve function re-initializes the queryset using EnterpriseCustomerUser